src/py/Pipfile

```diff
@@ -14,6 +14,7 @@ requests = "*"
 SQLAlchemy = "*"
 flask-cors = "*"
 sqlalchemy-enum-tables = {editable = true, path = "./third-party/sqlalchemy-enum-tables"}
+SQLAlchemy-Utils = "0.36.8"
```

Comment:
do we need this here if we're only going to be using this module with the happy cli? The application itself doesn't need this package right?

Reply:
It needs to be in the container (see comment above)

Reply:
cool